dist/formats/placeholder/frontend/schema.json

```diff
@@ -40,6 +40,10 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
+        "analytics_identifier": {
```

Comment:
Should this be present in the frontend schema? Sounds like it only needs to be in the links themselves, in which case it should be added to FrontendSchemaGenerator.INTERNAL_PROPERTIES.